core/src/components/segment-button/segment-button.ios.scss

```diff
@@ -22,9 +22,9 @@
   --indicator-transform: none;
   --transition: #{$segment-button-ios-transition};
   --padding-top: 0;
-  --padding-end: 13px;
+  --padding-end: 7px;
   --padding-bottom: 0;
-  --padding-start: 13px;
+  --padding-start: 7px;
```

Comment:
Does changing default values for CSS variables count as a breaking change? 🤔

Reply:
In the past we haven't considered it to be a breaking change since we update the colors for the spec all of the time. In this case it would probably be more of a breaking change for them if we don't update these. The reason for the update is because the text is now completely surrounded by the padding where it wasn't before, see this image:

![image](https://github.com/ionic-team/ionic-framework/assets/6577830/72b1cc62-8a72-4df5-9fe4-901f24c74a91)

Reply:
I am actually going to revert this padding change because this doesn't actually get us closer to native, it is the opposite.